src/util/__tests__/importText.ts

```diff
@@ -572,3 +572,17 @@ it('decode HTML entities', () => {
   - one & two
   - three < four`)
 })
+
+// related: https://github.com/cybersemics/em/issues/1008
+it('do not parse as html when value has tags inside indented text', () => {
+
+  expect(importExport(`
+  - a
+    - b
+    - <li>c</li>
+  `).trim())
```

Comment:
Let's remove the `trim`; I think in all our tests we do an exact match on the `importExport` result.

Reply:
Sure.